packages/ckeditor5-engine/src/conversion/downcasthelpers.js

```diff
@@ -1670,6 +1896,247 @@ function prepareDescriptor( highlightDescriptor, data, conversionApi ) {
 	return descriptor;
 }
 
+// Creates a function that checks a single differ diff item whether it should trigger reconversion.
+//
+// @param {Object} model A normalized `config.model` converter configuration.
+// @param {String} model.name The name of element.
+// @param {Array.<String>} model.attributes The list of attribute names that should trigger reconversion.
+// @param {Boolean} [model.children] Whether the child list change should trigger reconversion.
+// @returns {Function}
+function createChangeReducerCallback( model ) {
+	return ( node, change ) => {
+		if ( !node.is( 'element', model.name ) ) {
+			return null;
+		}
+
+		if ( change.type == 'attribute' ) {
+			if ( !model.attributes.includes( change.attributeKey ) ) {
+				return null;
+			}
+		} else {
+			if ( !model.children ) {
+				return null;
+			}
+		}
+
+		return [ node ];
```

Comment:
Why `null` or single-item array instead of \`true\` \`false\`?

Reply:
I see this function is used as default parameter later in the code. Will there be cases when the user will write this function on their own, and then the function would return bigger arrays?

Reply:
Or is this for future purposes? Or artifact from the past :)

Reply:
This was for future purposes but in a recent spike, I found a better way to handle it. In the next PoC it's true/false https://github.com/ckeditor/ckeditor5/compare/ck/10294/reconversion...ck/10381-imperative-reconversion